src/plugins/data/common/search/aggs/buckets/_terms_other_bucket_helper.test.ts

```diff
@@ -145,7 +146,7 @@ const nestedTermResponse = {
               { key: '__missing__', doc_count: 1430 },
             ],
           },
-          key: 'US',
+          key: 'US-with-dash',
```

Comment:
:information_source: This test change makes the old implementation fail, since it contains the dash.